builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -1233,22 +1312,12 @@ def loadServerlessConfig() {
 		sh "cp _config/serverless-go.yml ./serverless.yml"
 	}
 
-  if (service_config['event_source_dynamodb'] || service_config['event_source_sqs']) {
-    sh "cp _config/aws-events-policies/custom-policy.yml ./policyFile.yml"
-  }
-
-  if (!service_config['event_source_dynamodb'] || !service_config['event_source_sqs'] || !service_config['event_source_s3'] || !service_config['event_source_kinesis']) {
-    removeEventResources ()
-  }
+	if (service_config['event_source_dynamodb'] || service_config['event_source_sqs'] || service_config['event_source_dynamodb'] || service_config['event_source_s3'] || ( configLoader.TVAULT && configLoader.TVAULT.IS_ENABLED )) {
```

Comment:
What's the purpose of checking this flag: `TVAULT.IS_ENABLED` here?

Reply:
Currently we need the custom roles only for event enabled and vault enabled. So added the check